### ISSUE TITLE
[pytorch/ppc64le] test_cpp_api_extension test fails 

### ISSUE THREAD

**Issue by avmgithub** [pytorch/pytorch#12048]
Torch version : torch (1.0.0a0+dfa03e9)
RH 7.5

When running (python run_test.py -i cpp_extensions)  I get a failure like : 

pytorch/test/cpp_extensions/cpp_api_extension.cpp:9:21: error: ‘torch::nn’ has not been declared
 struct Net : torch::nn::Module {
                     ^
pytorch/test/cpp_extensions/cpp_api_extension.cpp:9:25: error: expected ‘{’ before ‘Module’
 struct Net : torch::nn::Module {

The solution it seems is to add : 

#include <torch/nn/module.h>
#include <torch/nn/modules/linear.h>
#include <torch/nn/modules/batchnorm.h>
#include <torch/nn/modules/dropout.h>

in  pytorch/test/cpp_extensions/cpp_api_extension.cpp

The build log shows the following below  (edited out some directories)

 c++ -MMD -MF cpp_api_extension.o.d -DTORCH_EXTENSION_NAME=cpp_api_extension -DTORCH_API_INCLUDE_EXTENSION_H -I/   ...   -isystem /miniconda/lib/python3.6/site-packages/torch/lib/include -isystem /miniconda/lib/python3.6/site-packages/torch/lib/include/torch/csrc/api/include -isystem /miniconda/lib/python3.6/site-packages/torch/lib/include/TH -isystem /miniconda/lib/python3.6/site-packages/torch/lib/include/THC -isystem /miniconda/include/python3.6m -fPIC -std=c++11 -UTORCH_API_INCLUDE_EXTENSION_H -c /builder/jenkins/pytorch/pytorch/test/cpp_extensions/cpp_api_extension.cpp -o cpp_api_extension.o
FAILED: cpp_api_extension.o

**Comment by avmgithub** [comment]
This looks to have been caused by a bad environment or corrupt pytorch installation. Closing for now.
